variable_set_test.go

```diff
@@ -190,16 +191,83 @@ func TestVariableSetsDelete(t *testing.T) {
 	})
 }
 
+func TestVariableSetsApplyToAndRemoveFromWorkspaces(t *testing.T) {
+	client := testClient(t)
+	ctx := context.Background()
+
+	orgTest, orgTestCleanup := createOrganization(t, client)
+	defer orgTestCleanup()
+
+	vsTest, vsTestCleanup := createVariableSet(t, client, orgTest, VariableSetCreateOptions{})
+	defer vsTestCleanup()
+
+	wTest1, wTest1Cleanup := createWorkspace(t, client, orgTest)
+	defer wTest1Cleanup()
+	wTest2, wTest2Cleanup := createWorkspace(t, client, orgTest)
+	defer wTest2Cleanup()
+
+	t.Run("with first workspace added", func(t *testing.T) {
```

Comment:
Would love to see some test cases for invalid options

Reply:
Cheers, will add some more test cases!

Reply:
Ok, some test cases added for invalid options in https://github.com/hashicorp/go-tfe/pull/375/commits/bd4f215a4bf912756de44e12b4a5aa3e33fbfd73 (and hilariously corrected via https://github.com/hashicorp/go-tfe/pull/375/commits/0fac7ae180594f9ecd4c2cde694ad2a23ea86f04).